roles/post-db-restore/tasks/main.yml

```diff
@@ -108,3 +108,16 @@
 - name: Copy private files from local to Site Factory
   shell: "drush -y rsync /tmp/{{ inventory_hostname }}/files/private/ @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%private/"
   notify: Clear site cache
+
+- name: Generate random password
+# See https://stackoverflow.com/a/45080709
+  set_fact:
+    random_pw: "{{ 999999999999999999999 | random | string + (lookup('pipe', 'date +%s%N')) | to_uuid() }}"
```

Comment:
This goes way past 11.

Reply:
![stackoverflow](https://user-images.githubusercontent.com/821106/44700104-478d4380-aa3c-11e8-8486-fef25e4cde73.jpg)